### PR TITLE
S02 stations mini cards

### PR DESCRIPTION
**Pull request by JulienOza**

### CODE REVIEW

File: frontend/src/components/Navbar/Navbar.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
+@import "../../variable.scss";
+
+.navbar_list {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  background: $gradient90deg;
+  height: 4rem;
```
Comment (Bravax) [inline frontend/src/components/Navbar/Navbar.scss:10]: Change the unit of measurement

File: frontend/src/components/Navbar/Navbar.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  width: 18rem;
```
Comment (Bravax) [inline frontend/src/components/Navbar/Navbar.scss:11]: Change the unit of measurement

File: frontend/src/components/Navbar/Navbar.scss [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  box-shadow: $boxShadow;
+  position: fixed;
+  bottom: 5%;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.navbar_element {
+  list-style: none;
+  width: 33.33%;
```
Comment (Bravax) [inline frontend/src/components/Navbar/Navbar.scss:23]: Change the unit of measurement

File: frontend/src/components/Navbar/Navbar.scss [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  height: 100%;
```
Comment (Bravax) [inline frontend/src/components/Navbar/Navbar.scss:24]: Change the unit of measurement

File: frontend/src/components/Navbar/Navbar.scss [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+}
+.navbar_element button {
+  all: unset;
+  cursor: pointer;
+}
+
+.navbar_element a,
+.navbar_element button {
+  width: 100%;
```
Comment (Bravax) [inline frontend/src/components/Navbar/Navbar.scss:33]: Idem

File: frontend/src/components/Navbar/Navbar.scss [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  height: 100%;
```
Comment (Bravax) [inline frontend/src/components/Navbar/Navbar.scss:34]: Idem

File: frontend/src/components/Navbar/Navbar.scss [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.navbar_element_middle {
+  border-left: $borderDark;
+  border-right: $borderDark;
+}
+
+.navbar_img {
+  height: auto;
+  width: 50%;
```
Comment (Bravax) [inline frontend/src/components/Navbar/Navbar.scss:46]: Idem

File: frontend/src/components/StationMiniCard/StationMiniCard.jsx [hunk 8]
```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import NavbarStations from "../../assets/navbar_stations.svg";
+import "./StationMiniCard.scss";
+
+function StationMiniCard({ stations }) {
+  return (
+    <article className="station_mc">
```
Comment (Bravax) [inline frontend/src/components/StationMiniCard/StationMiniCard.jsx]: Article or section?

File: frontend/src/components/StationMiniCard/StationMiniCard.scss [hunk 9]
```diff
@@ -0,0 +1,52 @@
+@import "../../variable.scss";
+
+.station_mc {
+  box-sizing: border-box;
+  height: 5rem;
```
Comment (Bravax) [inline frontend/src/components/StationMiniCard/StationMiniCard.scss:9]: Change the unit of measurement

File: frontend/src/components/StationMiniCard/StationMiniCard.scss [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+  border-radius: 10px;
+  box-shadow: $boxShadow_inv;
+  display: grid;
+  grid-template-columns: 25% 35% 1fr 15%;
+  grid-template-rows: 1fr 55%;
+  align-items: center;
+}
+
+.station_mc_logo {
+  width: 2.5rem;
```
Comment (Bravax) [inline frontend/src/components/StationMiniCard/StationMiniCard.scss:24]: Change the unit of measurement

File: frontend/src/components/StationMiniCard/StationMiniCard.scss [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+  height: 2.5rem;
```
Comment (Bravax) [inline frontend/src/components/StationMiniCard/StationMiniCard.scss:25]: Idem

File: frontend/src/components/StationMiniCard/StationMiniCard.scss [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+  padding: 0.25rem 0.4rem;
+  background: $gradient90deg;
+  border-radius: 10px;
+  box-shadow: $boxShadow_inv;
+  grid-area: 1 / 1 / 3 / 2;
+  justify-self: center;
+}
+
+.plug-type_logo {
+  width: 2.2rem;
```
Comment (Bravax) [inline frontend/src/components/StationMiniCard/StationMiniCard.scss:35]: Idem

File: frontend/src/components/Navbar/Navbar.scss (hunk 3 above)
Comment (JulienOza) [reply frontend/src/components/Navbar/Navbar.scss:23]: needeed precisely 1/3rd proportion for this element

File: frontend/src/components/Navbar/Navbar.scss (hunk 4 above)
Comment (JulienOza) [reply frontend/src/components/Navbar/Navbar.scss:24]: same here

File: frontend/src/components/Navbar/Navbar.scss (hunk 5 above)
Comment (JulienOza) [reply frontend/src/components/Navbar/Navbar.scss:33]: s100% needed for these elements, other units won't work here

File: frontend/src/components/StationMiniCard/StationMiniCard.jsx (hunk 8 above)
Comment (JulienOza) [reply frontend/src/components/StationMiniCard/StationMiniCard.jsx]: Article changed for "li" elements mapped into an "ul"

File: frontend/src/components/StationMiniCard/StationMiniCard.scss (hunk 9 above)
Comment (JulienOza) [reply frontend/src/components/StationMiniCard/StationMiniCard.scss:9]: this value is fixed on purpose, rem is the good unit here.

File: frontend/src/components/StationMiniCard/StationMiniCard.scss (hunk 10 above)
Comment (JulienOza) [reply frontend/src/components/StationMiniCard/StationMiniCard.scss:24]: this value is fixed on purpose, rem is the good unit here.

File: frontend/src/components/StationMiniCard/StationMiniCard.scss (hunk 12 above)
Comment (JulienOza) [reply frontend/src/components/StationMiniCard/StationMiniCard.scss:35]: this value is fixed on purpose, rem is the good unit here.

**Review by ItsMatDev — APPROVED** [review]

File: frontend/src/components/Navbar/Navbar.jsx [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+  return (
+    <nav className="navbar_container">
+      <ul className="navbar_list">
+        <li className="navbar_element">
+          <Link to="/map" draggable="false">
+            <img
+              className="navbar_img"
+              src={NavbarMap}
+              draggable="false"
+              alt="map logo"
```
Comment (Yoann-Fortin) [inline frontend/src/components/Navbar/Navbar.jsx]: All images are same name

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

File: frontend/src/components/Navbar/Navbar.jsx (hunk 13 above)
Comment (JulienOza) [reply frontend/src/components/Navbar/Navbar.jsx]: corrected !

**Review by SebSamPro — APPROVED** [review]

**Review by SebSamPro — APPROVED** [review]

**Review by Bravax — APPROVED** [review]

**Pull request merged**